deny.toml

```diff
@@ -1,10 +1,14 @@
+[graph]
+exclude = ["imap-codec-bench"]
```

Comment:
Oh, awesome. I was thinking about doing this but didn't find the option immediately. This is perfect.